src/game/client/components/menus.h

```diff
@@ -290,6 +290,8 @@ class CMenus : public CComponent
 	vec2 m_MousePos;
 	bool m_MouseSlow;
 
+	char m_NextServer[256];
```

Comment:
```suggestion
	char m_aNextServer[256];
```
Arrays are usually prefixed with `a` in Teeworlds/DDNet code.

Reply:
sure!